integration/spark-common-cluster-test/src/test/scala/org/apache/carbondata/cluster/sdv/generated/AlterTableTestCase.scala

```diff
@@ -1016,6 +1016,62 @@ class AlterTableTestCase extends QueryTest with BeforeAndAfterAll {
     }
   }
 
+
+  test("AlterTable Change Sort Scope 1") {
+    sql("DROP TABLE IF EXISTS t1")
+    sql(s"CREATE TABLE t1(age int, name string) STORED BY 'carbondata' TBLPROPERTIES" +
+        s"('sort_scope'='local_sort', 'sort_columns'='age')")
+    sql("ALTER TABLE t1 SET TBLPROPERTIES('sort_scope'='batch_sort')")
+
+    var sort_scope: String = ""
+    sql("DESCRIBE FORMATTED t1").collect().foreach(row => {
+      if (row.getAs[String]("col_name").equalsIgnoreCase("sort scope")) {
+        sort_scope = row.getAs[String]("data_type")
+      }
+    })
+    assert(sort_scope.equalsIgnoreCase("batch_sort"))
+    sql("DROP TABLE t1")
+  }
+
+  test("AlterTable Change Sort Scope 2") {
+    sql("DROP TABLE IF EXISTS t1")
+    sql(s"CREATE TABLE t1(age int, name string) STORED BY 'carbondata' TBLPROPERTIES" +
+        s"('sort_columns'='')")
+    sql("ALTER TABLE t1 SET TBLPROPERTIES('sort_scope'='local_sort')")
```

Comment:
Can we response a new exception of "set sort scope error" when sort_columns is empty
it's more meaningful and warn user